GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -1002,6 +1064,47 @@ + (void)proxyAppDelegate:(id<GULApplicationDelegate>)appDelegate {
   }
 }
 
++ (void)proxySceneDelegateWithClass:(Class)sceneDelegateClass API_AVAILABLE(ios(13.0)) {
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
```

Comment:
It looks like `dispatch_once` will allow only the first scene delegate to be swizzled, so other will be ignored.

Reply:
Yeah, good catch!